packages/client-account/src/AccountClientInstance.ts

```diff
@@ -22,9 +22,12 @@ export class AccountClientInstance extends Context.Tag(
  * @since 1.0.0
  * @category constructors
  */
-export const makeAccountClientInstance = Effect.map(
+export const makeAccountClientInstance = Effect.flatMap(
   AccountClientInstanceConfig,
-  (config) => new AccountClient(config),
+  (config) =>
+    Effect.acquireRelease(Effect.succeed(new AccountClient(config)), (client) =>
+      Effect.sync(() => client.destroy()),
+    ),
 );
 
 /**
```

Comment:
We can use Layer.scoped when constructing instance layer
```ts
export const AccountClientInstanceLayer = Layer.scoped(
  AccountClientInstance,
  makeAccountClientInstance,
);
```
in this way we can avoid introducing breaking change

additionally we can add `AccountClientInstanceLayerScoped` and `defaultLayerScoped` and other pairs, it is normal practice in effect ecosystem. if you want to control scope you use `scoped` version of layer.